packages/swapper/src/swappers/PortalsSwapper/utils/fetchPortalsTradeOrder.ts

```diff
@@ -75,12 +84,20 @@ type PortalsTradeOrderEstimateResponse = {
   }
 }
 
+export class PortalsError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.name = 'PortalsError'
+  }
+}
```

Comment:
Allows us to discriminate on Portals upstream error with `message` property explicitly, vs. other kind of failures, or Portals potentially responding without `message` which would be a breaking change and/or could totally happen currently under other scenarios we're not aware of.